homeassistant/components/apcupsd/sensor.py

```diff
@@ -66,9 +68,9 @@
     "nominv": ["Nominal Input Voltage", UNIT_VOLT, "mdi:flash"],
     "nomoutv": ["Nominal Output Voltage", UNIT_VOLT, "mdi:flash"],
     "nompower": ["Nominal Output Power", POWER_WATT, "mdi:flash"],
-    "nomapnt": ["Nominal Apparent Power", "VA", "mdi:flash"],
+    "nomapnt": ["Nominal Apparent Power", ELECTRICAL_VOLTAGE_AMPERE, "mdi:flash"],
```

Comment:
Why not `ELECTRICAL_CURRENT` ?

Reply:
Why not `ELECTRICAL_VOLT_AMPERE` as per https://en.wikipedia.org/wiki/Volt-ampere ? "Voltage" is not exactly the same as "Volt"

Reply:
Looks like you have answered your own question ✌